mantle/kola/tests/misc/zincati-deadend.go

```diff
@@ -0,0 +1,118 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package misc
+
+import (
+	"fmt"
+
+	"github.com/coreos/mantle/kola/cluster"
+	"github.com/coreos/mantle/kola/register"
+	"github.com/coreos/mantle/platform/conf"
+	"github.com/vincent-petithory/dataurl"
+)
+
+var graph = []byte(`{
+	"nodes": [
+	  {
+		"version": "${VERSION}",
+		"metadata": {
+		  "org.fedoraproject.coreos.scheme": "checksum",
+		  "org.fedoraproject.coreos.releases.age_index": "0",
+		  "org.fedoraproject.coreos.updates.deadend_reason": "https://github.com/coreos/fedora-coreos-tracker/issues/215",
+		  "org.fedoraproject.coreos.updates.deadend": "true"
+		},
+		"payload": "${PAYLOAD}"
+	  }
+	],
+	"edges": []
+  }`)
+
+var script = []byte(`#!/bin/bash
+  VERSION="$(/usr/bin/rpm-ostree status --json | jq '.deployments[0].version' -r)"
+  PAYLOAD="$(/usr/bin/rpm-ostree status --json | jq '.deployments[0].checksum' -r)"
+  sed -i 's/\"version\":.*/\"version\": \"'$VERSION'\"/g' /var/www/html/v1/graph
+  sed -i 's/\"payload\":.*/\"payload\": \"'$PAYLOAD'\"/g' /var/www/html/v1/graph`)
+
+var baseurl = []byte(`[cincinnati]
+base_url = "http://127.0.0.1:9001"`)
+
+var updates = []byte(`updates.enabled = true`)
+
+func init() {
+	register.RegisterTest(&register.Test{
+		Name:        "fcos.zincati.deadend.motd-info",
+		Run:         checkDeadEndReleaseInformation,
+		ClusterSize: 1,
+		Tags:        []string{"fcos", "zincati"},
+		UserDataV3: conf.Ignition(fmt.Sprintf(`{
+    "ignition": {"version": "3.0.0"},
+	"storage": {
+		"files": [
+		  {
+			"path": "/etc/zincati/config.d/99-cincinnati-baseurl.toml",
+			"contents": {
+			  "source": %q
+			}
+		  },
+		  {
+			"path": "/etc/zincati/config.d/99-enable-updates.toml",
+			"contents": {
+			  "source": %q
+			}
+		  },
+		  {
+			"path": "/var/www/html/v1/graph",
+			"contents": {
+			  "source": %q
+			}
+		  },
+		  {
+			"path": "/usr/local/bin/extract-dead-end.sh",
+			"contents": {
+			  "source": %q
+			},
+			"mode": 493
+		  }
+		]
+	  },
+	"systemd": {
+		"units": [
+		  {
+			"dropins": [
+			  {
+				"contents": "[Service]\nEnvironment=ZINCATI_VERBOSITY=\"-vvvv\"",
+				"name": "verbose.conf"
+			  }
+			],
+			"name": "zincati.service"
+		  },
+		  {
+			"contents": "[Unit]\nDescription=nginx Server\nAfter=network-online.target\nRequires=network-online.target\n[Service]\nRestart=on-failure\nExecStartPre=-/usr/bin/podman pull docker.io/nginx:latest\nExecStartPre=/usr/local/bin/extract-dead-end.sh\nExecStart=/usr/bin/podman run --name nginx \\\n                      -p 9001:80 \\\n                      -v /var/www/html:/usr/share/nginx/html \\\n                      nginx\nExecStop=-/usr/bin/podman stop nginx\n[Install]\nWantedBy=multi-user.target\n",
```

Comment:
For readability purposes, is it possible define the `contents:` in a var and then encode it?  Similar to what you've done for `baseurl`, `updates`, etc